util/src/harvey/cmd/build/paths.go

```diff
@@ -11,9 +11,6 @@ import (
 
 func init() {
 	harvey = os.Getenv("HARVEY")
-	if harvey != "" {
```

Comment:
I'm not so sure about this. I think the intent is that if you are running from a not-git place it will work. I'm actually willing to yank this whole function, it assumes you have git installed. Andy why can't people just set PATH and HARVEY without crutches?

I don't know enough to LGMT.